e2e/schematics/jest.test.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  newProject,
+  runCLI,
+  newLib,
+  copyMissingPackages,
+  updateFile,
+  readJson,
+  runCommand
+} from '../utils';
+
+describe('Jest', () => {
+  beforeAll(() => {
+    newProject();
+    runCLI('generate jest', {
+      silenceError: true
```

Comment:
This is part of the hack below it. There's nothing published under `@nrwl/builders` so the install fails. It can be removed once we publish the initial version.